src/main/java/net/dries007/tfc/client/ClientEventHandler.java

```diff
@@ -70,14 +71,7 @@
 import net.minecraftforge.fml.event.lifecycle.FMLClientSetupEvent;
 import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;
 import net.minecraftforge.registries.ForgeRegistries;
-
```

Comment:
why the import changes? what happened?

Reply:
I messed up and it turned importing the entity models into model.entity.* for a minute, which it seemed like you didn't want